templates/emby/0/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+emby:
+  image: emby/embyserver:latest
+  net: host
```

Comment:
Also, if you  check the [current docker installation instructions][instructions] you can see that after emby is installed you need to run a command in the machine host to start emby. 

I'm not sure at all, but I think that's not what we expect when we create a rancher stack (IMHO).

I've been installing emby with docker for a while and the old way works like a charm.

[instructions]: https://emby.media/linux-server.html